samples/TcpClient_NarodMon/app/application.cpp

```diff
@@ -51,6 +51,7 @@ bool nmOnReceive(TcpClient& client, char *buf, int size)
 	// debug msg
 	debugf("nmOnReceive");
 	debugf("%s", buf);
+	return true;
```

Comment:
For reference: https://github.com/SmingHub/Sming/blob/develop/Sming/SmingCore/Network/TcpClient.cpp#L126